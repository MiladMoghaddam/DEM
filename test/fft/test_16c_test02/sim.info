{'cmdline': ['../../run-sniper',
             '-n',
             '16',
             '-c',
             'gainestown',
             '-c',
             'noc',
             '--roi',
             '-d',
             './test_16c_test02',
             '-speriodic-stats:10000:0',
             '-senergystats:10000',
             '-g',
             '--perf_model/core/frequency=2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2',
             '--',
             './fft',
             '-p',
             '16'],
 'git_revision': '593709afd5ac1a5756c123e74e437122d9b328a1',
 'host': 'sealion',
 'pin_version': 71313,
 'rusage': (40.804,
            9.728,
            120932,
            0,
            0,
            0,
            12101641,
            0,
            0,
            0,
            84824,
            0,
            0,
            0,
            55567,
            195150),
 'snipercmd': ['bash',
               '-c',
               '/home/milad/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /home/milad/sniper/lib/pin_sim -c /home/milad/sniper/config/base.cfg --general/total_cores=16 --general/output_dir=/home/milad/sniper/test/fft/test_16c_test02 --config=/home/milad/sniper/config/nehalem.cfg --config=/home/milad/sniper/config/gainestown.cfg --config=/home/milad/sniper/config/noc.cfg --perf_model/core/frequency=2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2 -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/milad/sniper/test/fft/test_16c_test02/sim.scripts.py -g --hooks/script0args= -- ./fft -p 16'],
 't_elapsed': 64.92771100997925,
 't_start': 1484078996.070468,
 'tracecmds': [],
 'user': 'milad',
 'vmem': 2334834688L}
