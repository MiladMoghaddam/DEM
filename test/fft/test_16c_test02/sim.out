                                     | Core 0     | Core 1     | Core 2     | Core 3     | Core 4     | Core 5     | Core 6     | Core 7     | Core 8     | Core 9     | Core 10    | Core 11    | Core 12    | Core 13    | Core 14    | Core 15   
  Instructions                       |      81129 |      69310 |      69269 |      69373 |      69343 |      69272 |      69322 |      69401 |      69285 |      69115 |      69222 |      69237 |      69435 |      69289 |      69088 |      69154
  Cycles                             |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128 |     541128
  IPC                                |       0.15 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13 |       0.13
  Time (ns)                          |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564 |     270564
  Idle time (ns)                     |     218229 |     224666 |     226434 |     222024 |     220175 |     224029 |     223475 |     225096 |     223847 |     223010 |     224843 |     225879 |     223997 |     222318 |     222175 |     225145
  Idle time (%)                      |      80.7% |      83.0% |      83.7% |      82.1% |      81.4% |      82.8% |      82.6% |      83.2% |      82.7% |      82.4% |      83.1% |      83.5% |      82.8% |      82.2% |      82.1% |      83.2%
Branch predictor stats               |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num correct                        |       4515 |       2935 |       2935 |       2929 |       2894 |       2922 |       2925 |       2921 |       2921 |       2856 |       2889 |       2926 |       2922 |       2927 |       2864 |       2900
  num incorrect                      |        361 |        322 |        318 |        329 |        336 |        322 |        325 |        333 |        324 |        341 |        335 |        318 |        334 |        321 |        330 |        320
  misprediction rate                 |      7.40% |      9.89% |      9.78% |     10.10% |     10.40% |      9.93% |     10.00% |     10.23% |      9.98% |     10.67% |     10.39% |      9.80% |     10.26% |      9.88% |     10.33% |      9.94%
  mpki                               |       4.45 |       4.65 |       4.59 |       4.74 |       4.85 |       4.65 |       4.69 |       4.80 |       4.68 |       4.93 |       4.84 |       4.59 |       4.81 |       4.63 |       4.78 |       4.63
TLB Summary                          |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  I-TLB                              |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |       7716 |       6170 |       6163 |       6196 |       6216 |       6179 |       6193 |       6221 |       6193 |       6187 |       6197 |       6186 |       6245 |       6207 |       6193 |       6193
    num misses                       |          5 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29
    miss rate                        |      0.06% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.46% |      0.47% |      0.47% |      0.47%
    mpki                             |       0.06 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42
  D-TLB                              |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |      51765 |      46748 |      46728 |      46776 |      46763 |      46734 |      46752 |      46786 |      46738 |      46671 |      46713 |      46715 |      46803 |      46734 |      46662 |      46684
    num misses                       |         74 |         48 |         48 |         48 |         48 |         48 |         48 |         48 |         48 |         49 |         48 |         48 |         48 |         48 |         48 |         48
    miss rate                        |      0.14% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10%
    mpki                             |       0.91 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.71 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69
  L2 TLB                             |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |         79 |         77 |         77 |         77 |         77 |         77 |         77 |         77 |         77 |         78 |         77 |         77 |         77 |         77 |         77 |         77
    num misses                       |         51 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76
    miss rate                        |     64.56% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     97.44% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70%
    mpki                             |       0.63 |       1.10 |       1.10 |       1.10 |       1.10 |       1.10 |       1.10 |       1.10 |       1.10 |       1.10 |       1.10 |       1.10 |       1.09 |       1.10 |       1.10 |       1.10
Cache Summary                        |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  Cache L1-I                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |       7716 |       6170 |       6163 |       6196 |       6216 |       6179 |       6193 |       6221 |       6193 |       6187 |       6197 |       6186 |       6245 |       6207 |       6193 |       6193
    num cache misses                 |        215 |        265 |        266 |        267 |        275 |        266 |        268 |        270 |        268 |        275 |        270 |        264 |        268 |        268 |        274 |        267
    miss rate                        |      2.79% |      4.29% |      4.32% |      4.31% |      4.42% |      4.30% |      4.33% |      4.34% |      4.33% |      4.44% |      4.36% |      4.27% |      4.29% |      4.32% |      4.42% |      4.31%
    mpki                             |       2.65 |       3.82 |       3.84 |       3.85 |       3.97 |       3.84 |       3.87 |       3.89 |       3.87 |       3.98 |       3.90 |       3.81 |       3.86 |       3.87 |       3.97 |       3.86
  Cache L1-D                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |      52055 |      46852 |      46829 |      46880 |      46864 |      46833 |      46855 |      46890 |      46837 |      46764 |      46810 |      46812 |      46907 |      46833 |      46752 |      46779
    num cache misses                 |        577 |        324 |        318 |        332 |        345 |        324 |        320 |        330 |        319 |        339 |        327 |        314 |        324 |        316 |        346 |        330
    miss rate                        |      1.11% |      0.69% |      0.68% |      0.71% |      0.74% |      0.69% |      0.68% |      0.70% |      0.68% |      0.72% |      0.70% |      0.67% |      0.69% |      0.67% |      0.74% |      0.71%
    mpki                             |       7.11 |       4.67 |       4.59 |       4.79 |       4.98 |       4.68 |       4.62 |       4.75 |       4.60 |       4.90 |       4.72 |       4.54 |       4.67 |       4.56 |       5.01 |       4.77
  Cache L2                           |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |        864 |        649 |        652 |        665 |        694 |        660 |        645 |        672 |        657 |        675 |        668 |        640 |        652 |        648 |        694 |        659
    num cache misses                 |        646 |        575 |        570 |        585 |        605 |        574 |        572 |        584 |        571 |        598 |        581 |        562 |        576 |        568 |        601 |        578
    miss rate                        |     74.77% |     88.60% |     87.42% |     87.97% |     87.18% |     86.97% |     88.68% |     86.90% |     86.91% |     88.59% |     86.98% |     87.81% |     88.34% |     87.65% |     86.60% |     87.71%
    mpki                             |       7.96 |       8.30 |       8.23 |       8.43 |       8.72 |       8.29 |       8.25 |       8.41 |       8.24 |       8.65 |       8.39 |       8.12 |       8.30 |       8.20 |       8.70 |       8.36
  Cache L3                           |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |        732 |        639 |        638 |        657 |        679 |        650 |        629 |        662 |        641 |        665 |        652 |        630 |        636 |        638 |        675 |        646
    num cache misses                 |        446 |        207 |        173 |        276 |        419 |        321 |        241 |        197 |        384 |        313 |        219 |        223 |        208 |        325 |        321 |        289
    miss rate                        |     60.93% |     32.39% |     27.12% |     42.01% |     61.71% |     49.38% |     38.31% |     29.76% |     59.91% |     47.07% |     33.59% |     35.40% |     32.70% |     50.94% |     47.56% |     44.74%
    mpki                             |       5.50 |       2.99 |       2.50 |       3.98 |       6.04 |       4.63 |       3.48 |       2.84 |       5.54 |       4.53 |       3.16 |       3.22 |       3.00 |       4.69 |       4.65 |       4.18
DRAM summary                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num dram accesses                  |        665 |          0 |          0 |          0 |        632 |          0 |          0 |          0 |        598 |          0 |          0 |          0 |        618 |          0 |          0 |          0
  average dram access latency (ns)   |      67.17 |        inf |        inf |        inf |      57.84 |        inf |        inf |        inf |      56.69 |        inf |        inf |        inf |      58.21 |        inf |        inf |        inf
  average dram queueing delay        |      13.74 |       0.00 |       0.00 |       0.00 |       4.41 |       0.00 |       0.00 |       0.00 |       3.26 |       0.00 |       0.00 |       0.00 |       4.78 |       0.00 |       0.00 |       0.00
  average dram bandwidth utilization |      2.00% |      0.00% |      0.00% |      0.00% |      1.00% |      0.00% |      0.00% |      0.00% |      1.00% |      0.00% |      0.00% |      0.00% |      1.00% |      0.00% |      0.00% |      0.00%
Coherency Traffic                    |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num loads from dram                |         25 |         30 |         28 |         31 |         48 |         31 |         42 |         30 |         46 |         45 |         29 |         44 |         38 |         47 |         36 |         35
  num loads from dram cache          |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from remote cache        |         46 |         70 |         20 |         80 |        129 |         74 |        108 |         66 |         88 |         96 |         88 |         89 |         50 |        115 |         92 |        122
